app/api/events.py

```diff
@@ -132,6 +134,12 @@ def after_create_object(self, event, data, view_kwargs):
                                  status='accepted')
         save_to_db(role_invite, 'Organiser Role Invite Added')
         if data.get('original_image_url'):
+            try:
+                image_file = urllib.request.urlopen(data['original_image_url']).read()
+            except (urllib.error.HTTPError, urllib.error.URLError):
+                raise UnprocessableEntity(
+                    {'source': 'attributes/original-image-url'}, 'Invalid Image URL'
+                )
             uploaded_images = create_save_image_sizes(data['original_image_url'], 'event', event.id)
```

Comment:
This will make two api calls. 
do it like this:
``` Python
try: 
    uploaded_images = create_save_image_sizes(data['original_image_url'], 'event', event.id)
except (urllib.error.HTTPError, urllib.error.URLError):
              raise UnprocessableEntity(
                  {'source': 'attributes/original-image-url'}, 'Invalid Image URL'
             )
```

Test it though, since I am not very confident about Python exception handling.